.github/workflows/export_database_views.yml

```diff
@@ -0,0 +1,71 @@
+name: Export Moped Views to SQL
+
+on:
+  push:
+    paths:
+      - moped-database/migrations/**
+    branches-ignore:
+      - "main"
+      - "production"
+
+jobs:
+  publish-views:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Install SQLFluff
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y python3-pip
+          pip3 install sqlfluff
```

Comment:
Not saying we should but do you think there is any reason to be explicit about the version here?

Reply:
The version of pip? That's a good question. 

Normally, I totally get wanting to put guardrails on versions of libraries that are outside of the core python system like in a requirements.txt, but pip seems like something you always want the most recent version of. Also, since this is coming out of the ubuntu repos, there's really no guarantee that older versions of pip are available -- in fact, I think one should bank on stale packages being removed pretty quick. It's been my experience that linux distribution repos (ubuntu, debian are the ones i know the best) don't have that npm-like, or the pip-like, feature/system of having everything every version available forever.

Reply:
sorry, GH's UI isn't making it easy to see but this comment was on the sqlfluff line. I don't really think this is a big concern but wanted to mention it!

Reply:
Oh, totally - thank you Mike for clarifying that. That was me misunderstanding; it makes a lot of sense what you're suggesting. I just don't see any way that could hurt, and even more, it feels like good practice. I bet that `pip3` line can specify the version somehow too, so it'd be easy or we could use redirection to get it on the workflow VM. I'll do my best to PR this in soon.